src/terms/hartree.jl

```diff
@@ -11,22 +11,26 @@ For the Coulomb potential with periodic boundary conditions, this is rather
 where G is the Green's function of the periodic Laplacian with zero
 mean (-Δ G = sum_{R} 4π δ_R, integral of G zero on a unit cell).
 """
-struct Hartree end
-(::Hartree)(basis) = TermHartree(basis)
+struct Hartree
+    scaling::Real  # to scale by an arbitrary factor (useful for exploration)
+end
+Hartree() = Hartree(1)
+(H::Hartree)(basis) = TermHartree(basis, H.scaling)
 
 struct TermHartree <: Term
     basis::PlaneWaveBasis
+    scaling::Real
```

Comment:
Scaling might be a slight misnomer (sounds like it might not be a factor). How about `prefactor` ?

Reply:
So we settle on scaling because we find no better name?

Reply:
As I said above, I don't like prefactor but scaling_factor is good if you want.

Reply:
Then I think I'd prefer that.

Reply:
Sorry I missed the comment below.